### DISCUSSION TITLE
Docs appear in "Recent Posts"

### DISCUSSION THREAD

**Discussion by MarkLagodych** [razonyang/hugo-theme-bootstrap#464] [Q&A] [answered]
I am trying to create a home page.
I created `home.md` in the `content` folder (because `_index.md` does not appear at all) and specified `url = '/'`.
I also have `posts`, `docs` and `tutorials` folders in `content`. 
Posts have layout 'posts'. Docs and tutorials have layout 'docs'.
On the right side in the "Recent Posts" panel I get not only posts from `posts` but also from `docs` and `tutorials`. How can I fix that?

**Comment by razonyang** [comment, marked as answer]
That's expected, the `recent posts` includes all pages of `mainSections` on the homepage.

**Reply by MarkLagodych** [reply]
Thank you!
